Test/task10/task10_solution.py

```diff
@@ -0,0 +1,27 @@
+def decorator(*d_args, **d_kwargs):
+    def real_decorator(func):
+        def wrapper(*args, **kwargs):
+            print((d_args, d_kwargs))
+            return func(*args, **kwargs)
+        return wrapper
+    return real_decorator
+
+
+@decorator(1, 2, 3, [1, 2, 3], 'one', 'two', 'three', one=1, two=2, three=3)
+def identity(x):
+    return x
```

Comment:
Should be print here as well
```suggestion
def identity(x):
    print(x)
    return x
```